backend/service/aws/aws.go

```diff
@@ -232,6 +234,10 @@ type regionalClient struct {
 	sts         stsClient
 }
 
+func (r *regionalClient) Config() *aws.Config {
```

Comment:
I know its basic but lets add a simple test to assert this functionality

Reply:
Added a new test.